### DISCUSSION TITLE
Blendr Network x Akash

### DISCUSSION THREAD

**Discussion by piemanny** [akash-network/community#553] [Integrations]
Blendr is introducing a new decentralized network that uses unused GPU resources all over the world, in order to give access to high-performance computing. Blendr Network offers a unique plug & play system called 'BlendrBox' which allows users to power the network from anywhere in the world whilst being rewarded. https://x.com/BlendrNetwork/status/1777335000347574601 https://x.com/BlendrNetwork/status/1774116764705751333

With regards with Akash, we would like to integrate Akash into our network by the below:

- We can use our partner's idle gpu resources and they can use ours.
- We can provide our gpu resources to AI projects.
- We can provide our gpu resources to movie production and content creation companies.
- We can provide our gpu resources to cloud gaming companies.
- We can list any idle gpu resources on our marketplace and payouts will be on usage basis.
- Possibly ways of utilising our hardware for Akash

The varying complexities can be discussed in the group as I think they are a number of moving parts but essentially the Blendr Core team would love to explore what is exactly possible, and I am trying to facilitate that.

**Comment by ghost** [comment]
Hey there! 

So do you want the resources on akash to be available on blendr or you want blender resources to be available on akash ?

**Comment by piemanny** [comment]
We would like to explore both scenarios.
